test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go

```diff
@@ -924,3 +1328,50 @@ func getTunedStructuredData(profile *performancev2.PerformanceProfile) *ini.File
 	Expect(err).ToNot(HaveOccurred())
 	return cfg
 }
+
+func getCPUs(numaCoreSiblings map[int]map[int][]int, coreKey int, cpu *string, sep string) {
```

Comment:
getCPUs and getRanges both need a descriptive name

Reply:
Fixed it in the latest commit